packages/schemas/src/validator.ts

```diff
@@ -90,11 +105,20 @@ export function validatePlatformSchema(schema: Schema): string {
     }
 }
 
+/**
+ * Add/register a schema
+ * @param schema Schema
```

Comment:
FYI: The types are documented automatically from the code with TypeDoc. So you just need `@param name [description]`

Reply:
Ah right, will fix that